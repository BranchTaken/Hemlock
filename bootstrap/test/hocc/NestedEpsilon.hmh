# The two conflicting Îµ reductions should be attributed to the start state.
hocc
    token Tb
    token EOI
    start S ::=
      | A EOI
    nonterm A ::= B C
    nonterm B ::= Tb
    nonterm C ::=
      | D
      | epsilon
    nonterm D ::=
      | epsilon
