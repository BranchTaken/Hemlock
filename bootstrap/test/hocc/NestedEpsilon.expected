hocc: Parsing "./NestedEpsilon.hmh"
hocc: Generating LR(1) specification
hocc: 0 precedences, 4 tokens, 6 non-terminals (1 start), 7 productions
hocc: LR(1) item set compatibility: lr1
hocc: Generating LR(1) item set closures (+^.=add/split/merge)++++++++
hocc: Generating 9 LR(1) states
hocc: 1 unresolvable conflict in 1 state (0 ‚ä•, 0 shift-reduce, 1 reduce-reduce)
hocc: 0 unreachable states
hocc: 0 remergeable states
hocc: Searching for unused precedences/tokens/non-terminals/productions
hocc: Generating text report
hocc: Writing "./hocc/NestedEpsilon.txt"
